### PR TITLE
Adding and enabling HEALTHCHECK

### PR DESCRIPTION
**Pull request by Leopere**
HEALTHCHECK in Docker is useful for ensuring uptime and enables self-healing mechanisms to kick in.  It's also useful when using a `depends_on` directive in docker-compose

### CODE REVIEW

File: chainquery/stuff/healthcheck.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 ## TODO: Implement this at some point it requires CURL in the container.
```
Comment (EnigmaCurry) [inline chainquery/stuff/healthcheck.sh]: remove this TODO

**Review by EnigmaCurry — APPROVED** [review]
LGTM

File: chainquery/stuff/healthcheck.sh (hunk 1 above)
Comment (Leopere) [reply chainquery/stuff/healthcheck.sh]: Fixed

**Comment by Leopere** [comment]
Confirmed to be working via Slack in conversation with @EnigmaCurry

**Pull request merged**